libs/backendless-console-api.js

```diff
@@ -451,18 +452,35 @@ class Backendless {
             delete column.columnId
             delete column.metaInfo
 
-            //we have to preserve the dataSize for the columns
-            if (column.dataSize && ['INT', 'TEXT'].includes(column.dataType)) {
+            //we preserve dataSize for STRING and BOOLEAN only
+            if (column.dataSize && ['DATETIME', 'INT','TEXT', 'DATA_REF', 'JSON',' POINT', 'DOUBLE',].includes(column.dataType)) {
```

Comment:
1. If we preserve it for string and boolean, would not it be better to check for these two types only rather than checking for any other types?
2. Why would Boolean have dataSize?
3.
```suggestion
            if (column.dataSize && !['STRING', 'FILE_REF'].includes(column.dataType)) {
```